test/basic-querying.test.js

```diff
@@ -220,6 +220,26 @@ describe('basic-querying', function() {
       });
     });
 
+    it('should query sorted desc by order integer field even though there' +
+       'is an async model loaded hook', function(done) {
+      User.observe('loaded', function(ctx, next) {
```

Comment:
The `User` object is global and shared by all other tests. As can be seen from [Travis CI logs](https://travis-ci.org/strongloop/loopback-datasource-juggler/builds/124841135), other tests start failing with this change in place.

Now I think it is actually a good idea to run all `find` tests with this `loaded` hook, but we need to do it slightly differently:

1) Setup the observer in `before` hook
2) Remove the observer in `after` hook

```js
describe('find', function() {
  before(function setupDelayingLoadedHook() {
    // ...
  });

  after(function removeDelayingLoadHook() {
    // ...
  });

  // individual test cases come here
  it(...);
  // ...
});